README.md

```diff
@@ -3,9 +3,16 @@
 ## Overview
 
 CharmHelpers provides an opinionated set of tools for building Juju
-charms that work together. In addition to basic tasks like interact-
-ing with the charm environment and the machine it runs on, it also
-helps keep you build hooks and establish relations effortlessly.
+charms that work together. In addition to basic tasks like interacting
+with the charm environment and the machine it runs on, it also helps
+keep you build hooks and establish relations effortlessly.
```

Comment:
This line doesn't read very well?  I'm not sure what it is saying.  Is it "... also helps you to build hooks and establish relations effortlessly"?

Reply:
Well, that was the original one - I just fixed formatting. I'll fix the comment itself as well.